cardigan/package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "start-storybook -s ./public -p 9001 -c .storybook",
     "build": "build-storybook -s ./public -c .storybook -o .dist",
-    "deploy": "aws s3 sync .dist/ s3://cardigan.wellcomecollection.org --only-show-errors --acl public-read"
+    "deploy": "aws s3 sync .dist/ s3://cardigan.wellcomecollection.org --only-show-errors --acl public-read",
+    "chromatic": "npx chromatic"
```

Comment:
fwiw I think you can remove `npx` here as `yarn` knows to run a command using the stuff in `node_modules`

Reply:
So just `"chromatic": "chromatic"` ?

Reply:
yep